informer/src/informer_service/informer_service_handler.rs

```diff
@@ -35,17 +35,23 @@ pub struct InformerService {
     client: Arc<Client>,
     event_receiver: Receiver<Events>,
     subscription_receiver: Receiver<Subscription>,
+    sub_cancellation_receiver: Receiver<SubscriptionId>,
```

Comment:
Should we add an additional channel?
How about the subscription channel to send and receive the "register" and "deregister" event?
Do you have any opinion about it?